### ISSUE TITLE
Tau XV25 Stealth Battlesuits // Tactical Drone count

### ISSUE THREAD

**Issue by Cleinor** [BSData/wh40k#1841]
**File/Catalogue:** 
TAU

**Description:** Detailed issue description.

Index sais: "This unit may be accompanied by up to 2 Tactical Drones"

Battlescribe allows 2 Tactical Drones per Shas'ui

**Comment by Cleinor** [comment]
This could effect other units also, like xv8 crisis

**Comment by Cleinor** [comment]
<img width="930" alt="tacdrone" src="https://user-images.githubusercontent.com/13948364/33427807-e5b1df08-d5c6-11e7-83a2-4c4691d9fd6f.PNG">

Source: Index Xenos 2 Page 60

**Comment by amis92** [comment]
@Cleinor I've marked is as "check required" because I had a feeling this was setup correctly, not because I wasn't sure if rules were as you presented.

Now I've checked, our files indeed don't allow more than 2 drones per team. I don't have time to check, but it's quite likely you're using Mobile Format files - which are completely separate and not supported any longer by previous maintainer. I suggest you switch your data source to our repo.
